SharedSources/MediaLibraryModel/MediaLibraryBaseModel.swift

```diff
@@ -44,6 +45,10 @@ protocol MLBaseModel: MediaLibraryBaseModel {
 }
 
 extension MLBaseModel {
+    var cellType: MediaCollectionViewCell.Type {
+        return MovieCollectionViewCell.self
```

Comment:
I'm not sure to understand this addition, what happens will with the GenreModel for example?

Reply:
It's a fallback for now until this is implemented

Reply:
exactly :)

Reply:
Since it won't always be a media, I think we should we consider renaming it so that it can be more generic.

What about`BaseCollectionViewCell`?

Reply:
oh interesting so I can't always set a media on the cell? so I guess the base type is a VLCMLFile ?

Reply:
>  I can't always set a media on the cell?

Not sure to understand what you mean here, for example, you cannot have `VLCMLMedia` in a genre category.

The base class of the VLCMediaLibraryKit models would be `VLCMLObject`.
`VLCMLFile` is different from the old MediaLibrary, it is files contained in a certain Media.